components/shared/AppSidebar/AppSidebar.tsx

```diff
@@ -62,6 +59,24 @@ export const AppSideBar = () => {
     .slice(0, 5)
     .sort((a, b) => new Date(b.updated_at).getTime() - new Date(a.updated_at).getTime());
 
+  interface GlobalEventHandlersEventMap {
+    workspaceUpdated: CustomEvent;
+  }
+
+  useEffectOnce(() => {
+    const mutateHandler = (event: Event) => {
```

Comment:
Registers a custom event handler for updating workspaces in the drop down. This event gets dispatched when a new workspace is created or an existing one is updated. It calls useSWR's `mutate()` to update the workspaces list.

Reply:
Why the `// eslint-disable-next-line no-console` eslint skip here? I don't see a console log. Leftover from debugging?

Reply:
I opted for the custom event @brandonroberts becuase I could use `mutate()` from `useSWRConfig`, but I wanted to use the [bound mutate](https://swr.vercel.app/docs/mutation#bound-mutate) instead.

Reply:
It's here @jpmcb `event instanceof CustomEvent && console.info(WORKSPACE_UPDATED_EVENT, event.detail);`

Reply:
It's here @jpmcb, `event instanceof CustomEvent && console.info(WORKSPACE_UPDATED_EVENT, event.detail);`

Reply:
Ahhhhh missed the console log after the `&&` in that statment 🤦🏼 👍🏼